packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -87,15 +82,6 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
     }
 
     protected get serializer(): IFluidSerializer {
```

Comment:
I'm not sure if this method be implemented at this layer at all. I.e. why do we need it & this._serializer?
Maybe we do, but not very clear when it should / can be used.

Reply:
This getter is accessible by derived classes whereas this._serializer is not, in order to prevent its use while garbage collecting which only makes sense in SharedObject. I think the way I've split this functionality across SharedObjectCore and SharedObject makes this more confusing.